page.html

```diff
@@ -1,30 +1,34 @@
-
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <link href="fun.css" rel="stylesheet" type = "text/css">
-    <script type="text/javascript" src="fun.js"> </script>
-
-    <title>Timer</title>
-  </head>
-  <body>
-
-<p id="timer">00:00:00</p>
-
-<button onclick="start()">Start</button> 
-<button onclick="stop()">Stop</button>
-<button onclick="reset()">Reset</<button>
-
-
-
-
-
-
-
-
-
 
+<head>
+  <meta charset="utf-8">
+  <link href="fun.css" rel="stylesheet" type="text/css">
+  <script type="text/javascript" src="fun.js"> </script>
+  <title>Timer</title>
+</head>
+
+<body>
+  <h1 id="title">Timer</h1>
+  <div id="layout">
+
+    <div id="stopwatchLayout">
+      <h2 id="stitle">Stopwatch</h2>
+      <p id="stopwatch">00:00:00</p>
+      <button id="but1" onclick="start()">Start</button>
+      <button id="but2" onclick="stop()">Stop</button>
+      <button id="but3" onclick="reset()">Reset</<button>
+    </div>
+
+    <div id="stopwatchLayout">
```

Comment:
CSS ids should NEVER be used more than once.

You need to read this: https://css-tricks.com/the-difference-between-id-and-class/